.vscode/general.code-snippets

```diff
@@ -95,13 +95,26 @@
         "prefix": "nginxconf",
         "body": [
             "if [[ -f /install/.nginx.lock ]]; then",
-            "\techo_progress_start \"Configuring nginx\"",
-            "\tbash /etc/swizzin/scripts/nginx/${1:app}.sh",
+            "\techo_progress_start \"Configuring nginx for ${1:app}\"",
+            "\t/etc/swizzin/scripts/nginx/${1:app}.sh",
             "\tsystemctl reload nginx",
-            "\techo_progress_done \"nginx configured\"",
+            "\techo_progress_done \"Nginx configured\"",
             "else",
-            "\techo_info \"${1:app} will run on port ${2:port}\"",
-            "fi"
+            "\techo_info \"${1:application} will be running on port ${2:443}\"",
+            "fi$0",
+        ],
+    },
+    "Remove an nginx config": {
+        "scope": "shellscript",
+        "prefix": [
+            "nginxrm"
         ],
+        "body": [
+            "if [[ -f /install/.nginx.lock ]]; then",
+            "\trm /etc/nginx/apps/${1:app}.conf",
+            "\tsystemctl reload nginx",
+            "fi$0",
+        ],
+        "description": "Adds a source to a swizzin function.\nPrepends the appropriate shellcheck declaration"
```

Comment:
This doesn't belong here -- code snippets are not the scope of a Trackarr PR

Reply:
Yeah this is already also somewhere else I believe